docs/whats-new/staging-release-notes.md

```diff
@@ -1,16 +1,39 @@
 # Staging release notes
 
+## Staging release 31 August 2022
+Frontend version: <version number in the environment> | Backend version: <version number in the environment>
+
+**New features** - **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Terminate accounts that are inactive for 30 days from creation</summary>
+
+TechPass will automatically terminate TechPass accounts that have not been used within 30 days from its creation date. The user will receive an email notification for this.
+
+<kbd>![email](../assets/images/whats-new/terminate-inactive-account.png)</kbd>
+</details>
+
+**Fixes** - **TechPass portal**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Warning message for expiring and expired certificates and secrets</summary>
+
+we have fixed a bug and now warning messages will be displayed for both expiring and expired certificates and secrets on the portal. Earlier, it was displayed only for expiring certificates and secrets.
+
+</details>
+
 ## Staging release 17 August 2022
-Frontend version:  | Backend version:   
+Frontend version: 1.0.0-20220808.0908 | Backend version: 1.27.8-220817.0220  
```

Comment:
i already see this info in https://docs.developer.tech.gov.sg/docs/techpass-user-guide/whats-new/staging-release-notes?id=staging-release-17-august-2022

you didn't pull the latest?

Reply:
i think i pulled the latest but agreed i could see the version on the staging docs but my code didn't have that info, so i just copied the info from the published content.